### PR TITLE
Ophys rig and session

### PR DESCRIPTION
**Pull request by saskiad**
major updates to ophys stuff

### CODE REVIEW

File: schemas/ophys/ophys_session.json [hunk 1]
```diff
@@ -59,219 +58,106 @@
...
-                "targeted_structure",
-                "targeted_ccf_coordinates",
-                "targeted_lab_coordinates",
-                "manipulator_coordinates",
-                "manipulator_angles"
-            ]
+            "required": ["fiber_name", "patch_cord_name"]
         },
-        "laser": {
+        "dectector": {
```
Comment (mekhlakapoor) [inline schemas/ophys/ophys_session.json:147]: fix detector typo

File: schemas/ophys/ophys_session.json [hunk 2]
```diff
@@ -59,219 +58,106 @@
...
+            "uniqueItems": true,
+            "title": "Coupling array"
         },
-        "coordinate_transform": {
-            "type": "string",
-            "title": "Coordinate transform",
-            "description": "Path to file that details the coordinate transform."
+        "detectors": {
+            "type": "array",
+            "itemsn": {"$ref": "#/$defs/dectector"},
```
Comment (mekhlakapoor) [inline schemas/ophys/ophys_session.json:81]: fix typo: "items": {"$ref": "#/$defs/detector"}
 (currently giving missing items definition error)

File: schemas/ophys/ophys_session.json [hunk 3]
```diff
@@ -284,8 +170,8 @@
         "project_id",
         "session_type",
         "rig_id",
-        "probes",
+        "patch_cords",
         "lasers",
-        "ccf_version"
+        "coupling_array"
     ]
 }
```
Comment (mekhlakapoor) [inline schemas/ophys/ophys_session.json:177]: add empty line at end of file

**Review by mekhlakapoor — CHANGES_REQUESTED** [review]
There's some minor typos so once those are addressed it looks good to merge

File: schemas/ophys/ophys_session.json (hunk 3 above)
Comment (saskiad) [reply schemas/ophys/ophys_session.json:177]: not sure what this means - just add an empty line? Is this something that needs to be there? I added one now - but let me know if I'm confused.

**Review by mekhlakapoor — APPROVED** [review]

**Pull request merged**